src/slack-model.jl

```diff
@@ -1,6 +1,29 @@
 export SlackModel, SlackNLSModel
 
-get_slack_ind(jl, ind) = map(b -> findfirst(x -> x == b, ind), intersect(jl, ind))
+function get_slack_ind(jl, ind)
+  if intersect(jl, ind) == []
+    return Int[]
+  end
+  return map(b -> findfirst(x -> x == b, ind), intersect(jl, ind))
+end
+
+"""
+    get_relative_indices(model)
+
+Return two sets of triplet with the relative indices of `jlow`, `jupp` and `jrng`
+within the set of linear and nonlinear indices.
+"""
+function get_relative_indices(model)
+  lin = model.meta.lin
```

Comment:
You're not saving much by defining this variable since you still write `model.meta.jlow`, etc., below. I suggest removing this variable.